src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,11 +5,34 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
```

Comment:
Don't begin class or method implementation with empty line. Remove the empty lines before the class declaration.